app/models/searchgov_document.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SearchgovDocument < ApplicationRecord
+  # SRCH-3134: web_document stores either:
+  # 1. the entire raw html of an html document, or
+  # 2. the full metadata and content of an application document of an application document.
```

Comment:
typo: "of an application document of an application document."

Reply:
Whoops, this is why I don't normally work in an office.  Fixed.